### PR TITLE
Change API responses to returning resources

### PR DESCRIPTION
**Pull request by kimong1**
For example
image

### CODE REVIEW

File: lib/users.js [hunk 1]
```diff
@@ -92,18 +92,20 @@ users.setDBPassword = async (userInfo, newPW) => {
...
       dbPassword: newPW
     }, {
       where: {
         username: userInfo.username
-      }
+      },
+      returning: true
     })
+    logger.info('DB password has set successfully', userInfo.username)
+    return updatedAccount[1][0]
```
Comment (greenc123) [inline lib/users.js:104]: What does the updated account look like? Why are you only accessing index 1?

File: lib/users.test.js [hunk 2]
```diff
@@ -96,32 +102,28 @@ describe('Sign up', () => {
...
-      password: 'password',
+      username: 'testuser',
+      password: 'abcd1234',
+      email: 'test.user@databases.com'
     }
-
-    await signUp(obj)
-
-    expect(mockCreateAccount.mock.calls.length).toEqual(1)
-    expect(mockCreateAccount.mock.calls[0][0].username).toEqual('Batman')
```
Comment (songz) [inline lib/users.test.js:121]: so you decided to only check input and output of the function? You don't want to check if the `createAccount` is getting called?

File: lib/users.js (hunk 1 above)
Comment (greenc123) [reply lib/users.js:104]: I figured it out from the test but could you add a comment for future engineers?

File: lib/users.test.js (hunk 2 above)
Comment (kimong1) [reply lib/users.test.js:121]: This way makes sure creatAccount is called

File: lib/users.js (hunk 1 above)
Comment (kimong1) [reply lib/users.js:104]: Okay I'll do tonight

**Pull request merged**